### DISCUSSION TITLE
Using FINN for Wiondos version of VIVAdo/Vitis

### DISCUSSION THREAD

**Discussion by AliMirhosseini** [Xilinx/finn#1065] [General]
Hi this is Ali, I'm new in FINN and having some questions about FINN and how to use it. I've read the "Getting start". 
1. I have Zynq-7020 SOC 
2. I'm working with windows version of Vivado 2018.2 ,2019.2 and 2023.2 versions 
3. I just need to creat the accelerator and will train, quentize and prune the Network by my self or using the Brevitas(there is no problem in this part)
the problem is, can i use the HLS_lib in windows version to creat the IP-core and use it as the accelerator by axi-stream in Vivado? 
without using pynq or petalinux? Imean acting with the accelrator like a FIR filter or someting like this? does the FINN HLS_lib sopurt this? 
actually i need a framwork to hellp me to buld a accelerator for CNN in the as fast as possible.

**Comment by noisyor** [comment]
Which version of Vivado did you use ? I am trying 2023.2 but getting vitis_hls not found in PATH error. Were you able to run it on your end with the 2023.2 version ? The startup page mentions using 2022.2 version.
